### PR TITLE
Benchmark against MySQL

### PR DESCRIPTION
**Pull request by christgf**
* MySQL connector dependency
* Spring MySQL profile
* Fixed broken migration script (sorry!)
* RuntimePropertyResolver can now resolve CSV values
* BenchmarkRunner now resolves and passes JVM arguments properly

### CODE REVIEW

File: battlecode/src/main/resources/db/migration/schema/V1.0.1__columnar.sql [hunk 1]
```diff
@@ -1,9 +1,9 @@
 CREATE TABLE IF NOT EXISTS columnar_properties (
     id                      BIGINT(32)      UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
-    value_a                 VARCHAR(255)    UNSIGNED NOT NULL,
```
Comment (christgf) [inline battlecode/src/main/resources/db/migration/schema/V1.0.1__columnar.sql]: implements CopyPasteable

File: battlecode/src/main/resources/application.yml [hunk 2]
```diff
@@ -38,4 +38,17 @@ logging:
...
+spring:
+  profiles: mysql
+
+  datasource:
+    url: jdbc:mysql://127.0.0.1:32768/battlecode_db?autoReconnect=true
+    username: root
+    password: 
+    driverClassName: com.mysql.jdbc.Driver
+    maxPoolSize: 8
+    connectTimeout: 1800
```
Comment (yavosh) [inline battlecode/src/main/resources/application.yml]: Do you need jpa settings for pg I use

```
    database-platform: org.hibernate.dialect.PostgreSQLDialect
    database: postgresql 
```

File: battlecode/src/test/java/org/battlecode/benchmark/BenchmarkRunner.java [hunk 3]
```diff
@@ -18,7 +16,7 @@ public static void main(String[] args) throws Exception {
         int forks = propertyResolver.getInteger("benchmark.forks", 1);
         int threads = propertyResolver.getInteger("benchmark.threads", 1);
         boolean shouldDoGc = propertyResolver.getBoolean("benchmark.gc", true);
-        String jvmArgs = propertyResolver.getProperty("benchmark.jvm.args");
+        String[] jvmArgs = propertyResolver.getCommaSeparated("benchmark.jvm.args");
```
Comment (yavosh) [inline battlecode/src/test/java/org/battlecode/benchmark/BenchmarkRunner.java]: name does not match getProperty perhaps getPropertyCommaSeparated or change both to getValue and getValueArray

Comment (christgf) [reply battlecode/src/test/java/org/battlecode/benchmark/BenchmarkRunner.java]: We've named the others getInteger, getBoolean, etc. In that sense, I think it is reasonably named. We need a convention, let's discuss at next standup.

File: battlecode/src/main/resources/application.yml (hunk 2 above)
Comment (christgf) [reply battlecode/src/main/resources/application.yml]: I guess not, but maybe pool size and other such properties could prove useful for benchmarks? Feel free to adjust as you see fit, maybe even create a PR at some point.

File: battlecode/src/test/java/org/battlecode/benchmark/BenchmarkRunner.java (hunk 3 above)
Comment (yavosh) [reply battlecode/src/test/java/org/battlecode/benchmark/BenchmarkRunner.java]: Then it should get getString for the first and getList or getArray for the second.

**Pull request merged**